lms/djangoapps/certificates/tests/test_models.py

```diff
@@ -14,7 +14,14 @@
     ExampleCertificateSet,
     CertificateHtmlViewConfiguration,
     CertificateTemplateAsset,
-    BadgeImageConfiguration)
+    BadgeImageConfiguration,
+    EligibleCertificateManager,
+    GeneratedCertificate,
```

Comment:
NIT: can we we also move `opaque_keys.edx.locator` to be in the correct alphabetical order.